index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<!---->
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+    integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Labrada:wght@300&family=Poppins&family=Roboto:wght@300&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="css/reset.css">
+  <link rel="stylesheet" href="css/style.css">
+
+  <title>Julio Vergara Portfolio</title>
+</head>
+
+<body>
+
+  <div id="container">
+    <div id="appbar">
+      <div id="time">3:33</div>
+      <div id="status">
+        <i class="fa-solid fa-signal"></i>
+        <i class="fa-solid fa-wifi"></i>
+        <i class="fa-solid fa-battery-full"></i>
+      </div>
+    </div>
+
+    <div id="toolbar">
+      <div id="logo-container">
+        <div id="logo">My Logo</div>
+      </div>
+      <div id="menu-container">
+        <nav>
+          <img id="menu" src="assets/Icons/Union.png" alt="">
+        </nav>
+      </div>
+    </div>
+  </div>
+
+  <header>
+    <div id="main-container">
+      <h1 id="main-title">Hey There I'm Julio</h1>
+      <h2 id="support-text">I'm a Software Developer</h2>
+      <p id="description">I can help you build a product , feature or website Look through some of my work and
+        experience!
+        If you like what you see and have a project you need coded,
+        don’t hesitate to contact me.</p>
+      <div id="social-media">
+        <i class="fa-brands fa-github icons"></i>
+        <i class="fa-brands fa-linkedin-in icons"></i>
+        <i class="fa-regular fa-hand-peace icons"></i>
+        <i class="fa-brands fa-twitter icons"></i>
+        <i class="fa-solid fa-m icons"></i>
+      </div>
```

Comment:
- [ ] Please inclose your icons in to a `ul` and `li` elements instead of just Plain `div` for semantic purpose. The functional advantage is that `divs` mean little on their own, whereas` ul` `li` explicitly mean "this is an un/ordered list of items." The term "semantics" refers to the way that you use the inherent meaning of existing structures to create explicit meaning.

Reply:
Done, Thank you!